mobile-app/cypress/integration/functional/wallet/auctions/auctions.spec.ts

```diff
@@ -384,108 +384,3 @@ context('Wallet - Auctions', () => {
     })
   })
 })
-
-context('Wallet - Auctions Feature Gated', () => {
```

Comment:
I think we can keep this test? Or maybe replace it with a different one? So we still have a test for feature gates

Reply:
I don't see any features where we can use this as of the moment. Any ideas? Can add this on this feature when it's already done --  https://github.com/DeFiCh/wallet/issues/2352

Reply:
What about adding e2e for the feature gates for loan payback using DFI (added in this [PR](https://github.com/DeFiCh/wallet/pull/2066))?

Reply:
Or can't we keep this here? Since we're intercepting the flags request.

Reply:
> Or can't we keep this here? Since we're intercepting the flags request.

Sure, we can keep the tests! I just modified one for it to pass. Since it's already removed in `Beta features` UI, I replaced it with setting the local storage manually

https://github.com/DeFiCh/wallet/pull/2435/files#diff-921bcc228ddc66f53efd9fc3c614f7a0158adba4256eb2d74acb35b34c17ede6R459